internal/command/node_command.go

```diff
@@ -93,6 +93,24 @@ func NewNodeCommand() *cobra.Command {
 				),
 			)
 
+			r.Route("/statistic", func(r chi.Router) {
```

Comment:
Why not to use a single handler config instance?

https://play.golang.org/p/Q8Q4EssIrIv